README.md

```diff
@@ -0,0 +1,186 @@
+<a id="readme-top"></a>
+
+<div align="center">
+  <br>
+  <h1 style="font-size: 40px; color: #ec5242">
+    <img style="width: 30px; padding-right: 3px;" src="src/logo/logo.svg" alt="logo icon">
+    <b>WebDev Academy</b>
+  </h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🎬 Representation of project](#representation)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<br>
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 WebDev Academy <a id="about-project"></a>
+
+**WebDev Academy** is an example of mobile first Online Educational Intitution website. It's purpose is to represent the institution, its different courses and the famous teachers from all around the world while giving the user a welcome feeling and a way to join the program.
+
+## 🛠 Built With <a id="built-with"></a>
+
+### Tech Stack <a id="tech-stack"></a>
+
+<details style="padding: 3px 0;">
+  <summary>Technology</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JAVASCRIPT</a></li>
+    </ul>
+</details>
+
+<details style="padding: 3px 0;">
+  <summary>Tools</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">VSCode</a></li>
+    <li><a href="https://git-scm.com/">Git</a></li>
+    <li><a href="https://www.behance.net/">Adobe Behance</a></li>
+  </ul>
+</details>
+<br/>
+
+<!-- Features -->
+### Key Features <a id="key-features"></a>
+
+- **Representation of The WebDev Academy**
+- **Mobile first responsive design**
+- **Giving a peak into different courses available**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a id="live-demo"></a>
+
+- An implementation of this portfolio is live on Github Pages(https://dev89s.github.io/webdev-inst/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎬 Representation <a id="representation"></a>
+
+- A representation of the project by Author himself. [Watch on loom](https://www.loom.com/share/bfaccfc8e55f49b8bc7c299d5e65aba6)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] While you did a great effort :+1: it is required for microverse to create a video with a maximum length of 5 minutes, which includes:
- A description of the project.
- A demo of the project features (different pages and different screen sizes).
- You should also highlight some interesting piece of code or something you built that you are very proud of.